.circleci/config.yml

```diff
@@ -0,0 +1,26 @@
+version: 2
+jobs:
+  deploy:
+    docker:
+      - image: circleci/python:3-node
+    steps:
+      - checkout
+      - run: yarn install --ignore-engines
+
+      # TODO: Maybe fix this? Or at least delete this comment
+      - run: CI=false yarn run build
+      - run: pip install awscli --upgrade --user
+
+      # I'm not sure why it installs here but it does...
```

Comment:
That's the default location of executable files in `pip install --user` command.